solidity/random-beacon/contracts/RandomBeacon.sol

```diff
@@ -1085,6 +1085,7 @@ contract RandomBeacon is IRandomBeacon, IApplication, Ownable, Reimbursable {
             // avoid blocking the future group creation.
             if (dkg.currentState() == DKG.State.AWAITING_SEED) {
                 dkg.notifySeedTimedOut();
+                dkg.complete();
```

Comment:
Why don't we call `complete()` from `notifySeedTimedOut`? Documentation to `BeaconDkg.notifySeedTimedOut` says "Notifies about the seed was not delivered and restores the initial DKG state (IDLE)" but it's in fact doing nothing other than checking the state:

https://github.com/keep-network/keep-core/blob/9675fe9f51cb4c63a73ae55f106a3454b3fbf643/solidity/random-beacon/contracts/libraries/BeaconDkg.sol#L277-L286

Reply:
Same about `notifyTimeout()`. All occurrences of `dkg.notifyTimeout()` and `dkg.notifySeedTimeout()` in `RandomBeacon` and `WalletRegistry` are followed by `dkg.complete()`.

Reply:
Yeah, I was thinking about it. I'm glad you brought it up! I'll update the code.

Reply:
91ce31d5b57fb98164be6a891310ad6527afdf05